### PR TITLE
refactor to call unsub on removelistener

### PR DESCRIPTION
**Pull request by ggazzo**
@sampaiodiego I think we should release a 1.0.0 version :x

### CODE REVIEW

File: packages/rocketchat-streamer/lib/ev.js [hunk 1]
```diff
@@ -7,15 +7,15 @@ EV = class EV {
...
 
 	emitWithScope(event, scope, ...args) {
-		if (this.handlers[event]) {
-			this.handlers[event].forEach((handler) => handler.apply(scope, args));
-		}
+		return this.handlers[event] && this.handlers[event].forEach(handler => handler.apply(scope, args));
+	}
+
+	listenerCount(event) {
+		return (this.handlers[event] || 0) && this.handlers[event].length;
```
Comment (sampaiodiego) [inline packages/rocketchat-streamer/lib/ev.js]: I think this should make more sense, wdyt?
```javascript
return (this.handlers[event] && this.handlers[event].length) || 0;

**Comment by ggazzo** [comment]
@sampaiodiego

**Comment by reyalpsirc** [comment]
@sampaiodiego will this solve the problem reported at https://forums.rocket.chat/t/unsubscribe-from-stream-room-messages/1500 ?

**Comment by ggazzo** [comment]
@reyalpsirc actually not, this code solve a frontend problem (we never ask to unsubscribe)

could you check if this "code" is right ?
image
because I think you should send `unsub` instead `nosub`

**Comment by reyalpsirc** [comment]
@ggazzo Sorry, it was a typo on my end. I actually call `unsub` and them I receive a `nosub`

**Comment by ggazzo** [comment]
🙄

**Comment by ggazzo** [comment]
@sampaiodiego could you review again please? <3

**Pull request merged**